package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "csbot",
 	"version": "0.12.1",
-	"lockfileVersion": 2,
+	"lockfileVersion": 3,
```

Comment:
Is there a benefit to using version 3 over version 2? I'm not familiar with the difference.

Reply:
Only difference is this is what NPM generated for us.